rotkehlchen/accounting/structures.py

```diff
@@ -337,7 +337,9 @@ class HistoryBaseEntry:
     # When we use this structure in blockchains can be used to specify addresses for example.
     # currently we use to identify the exchange name assigned by the user.
     location_label: Optional[str]
-    asset_balance: AssetBalance  # usd value starts 0: https://github.com/rotki/rotki/issues/3865
+    asset: Asset
+    amount: FVal
+    usd_value: FVal  # https://github.com/rotki/rotki/issues/3865
```

Comment:
Oh for balance it's fine to use it. AssetBalance together is too much.

Reply:
Okey, will got with balance here